hack/util.sh

```diff
@@ -403,6 +412,158 @@ function delete_large_and_empty_logs()
   find ${LOG_DIR} -name *.log -size 0 -exec echo Deleting {} because it is empty. \; -exec rm -f {} \;
 }
 
+######
+# start of common functions for extended test group's run.sh scripts
+######
+
+# exit run if ginkgo not installed
+function ginkgo_check_extended {
+    which ginkgo &>/dev/null || (echo 'Run: "go get github.com/onsi/ginkgo/ginkgo"' && exit 1)
+}
+
+# create extended.test binary to run extended tests
+function compile_extended {
+    # Compile the extended tests first to avoid waiting for OpenShift server to
+    # start and fail sooner on compilation errors.
+    echo "[INFO] Compiling test/extended package ..."
+    GOPATH="${OS_ROOT}/Godeps/_workspace:${GOPATH}" \
+	  go test -c ./test/extended -o ${OS_OUTPUT_BINPATH}/extended.test || exit 1
+    export GOPATH
+}
+
+# various env var and setup for directories and image related information
+function dirs_image_env_setup_extended {
+    export TIME_SEC=1000
+    export TIME_MIN=$((60 * $TIME_SEC))
+
+    export TEST_TYPE="openshift-extended-tests"
+    export TMPDIR="${TMPDIR:-"/tmp"}"
+    export BASETMPDIR="${TMPDIR}/${TEST_TYPE}"
+
+    if [[ -d "${BASETMPDIR}" ]]; then
+	remove_tmp_dir $TEST_TYPE
+    fi
+
+    mkdir -p ${BASETMPDIR}
+
+    # Use either the latest release built images, or latest.
+    if [[ -z "${USE_IMAGES-}" ]]; then
+	export USE_IMAGES='openshift/origin-${component}:latest'
+	if [[ -e "${OS_ROOT}/_output/local/releases/.commit" ]]; then
+	    export COMMIT="$(cat "${OS_ROOT}/_output/local/releases/.commit")"
+	    export USE_IMAGES="openshift/origin-\${component}:${COMMIT}"
+	fi
+    fi
+
+    export LOG_DIR="${LOG_DIR:-${BASETMPDIR}/logs}"
+    export ARTIFACT_DIR="${ARTIFACT_DIR:-${BASETMPDIR}/artifacts}"
+    export DEFAULT_SERVER_IP=`ifconfig | grep -Ev "(127.0.0.1|172.17.42.1)" | grep "inet " | head -n 1 | sed 's/adr://' | awk '{print $2}'`
+    export API_HOST="${API_HOST:-${DEFAULT_SERVER_IP}}"
+    mkdir -p $LOG_DIR $ARTIFACT_DIR
+}
+
+# env var setup for docker and build strategy related items
+function docker_strategy_env_setup_extended {
+    # use the docker bridge ip address until there is a good way to get the auto-selected address from master
+    # this address is considered stable
+    # used as a resolve IP to test routing
+    CONTAINER_ACCESSIBLE_API_HOST="${CONTAINER_ACCESSIBLE_API_HOST:-172.17.42.1}"
+
+    STI_CONFIG_FILE="${LOG_DIR}/stiAppConfig.json"
+    DOCKER_CONFIG_FILE="${LOG_DIR}/dockerAppConfig.json"
+    CUSTOM_CONFIG_FILE="${LOG_DIR}/customAppConfig.json"
```

Comment:
I don't see that this env variable is referenced anywhere, nor do i see where this file exists or is created?  (i know you moved it from @mfojtik 's test case, so maybe the question is on him, but it looks vestigial)